sdk/keyvault/Azure.Security.KeyVault.Keys/README.md

```diff
@@ -280,21 +280,21 @@ When deleting a key asynchronously before you purge it, you can await the `WaitF
 By default, this loops indefinitely but you can cancel it by passing a `CancellationToken`.
 
 ```C# Snippet:DeleteAndPurgeKeyAsync
-DeleteKeyOperation operation = client.StartDeleteKey("key-name");
+DeleteKeyOperation operation = await client.StartDeleteKeyAsync("key-name");
 
 // You only need to wait for completion if you want to purge or recover the key.
 await operation.WaitForCompletionAsync();
 
 DeletedKey key = operation.Value;
-client.PurgeDeletedKey(key.Name);
+await client.PurgeDeletedKeyAsync(key.Name);
```

Comment:
Same here. Be sure to actually update the code, then run `eng\Checks.ps1 -ServiceDirectory keyvault` from the root.

Reply:
There is no such ps1 in eng root. Is `Update-Snippets.ps1` ps1? I changed the sample code what from region "Snippet:DeleteAndPurgeKeyAsync", then run `eng\Update-Snippets.ps1` to update the readme code.